pkg/status/status.go

```diff
@@ -44,7 +43,7 @@ func GetStatus() (map[string]interface{}, error) {
 	stats["config"] = getPartialConfig()
 	stats["conf_file"] = config.Datadog.ConfigFileUsed()
 
-	platformPayload, err := new(platform.Platform).Collect()
+	platformPayload, err := getPlatformPayload()
```

Comment:
does the rest of the code correctly handle the `nil`value for `platformPayload`? Found this usage in a template which would break I think: https://github.com/DataDog/datadog-agent/blob/6.0.3/pkg/status/dist/templates/header.tmpl#L7 , may want to add a condition on `.platform` there

Reply:
Changed the return value to an empty map instead of `nil`. The default behaviour is `platform.SomethingMissing` printing `<no value>`. This would break if we instantiate the template passing the option `missingkey=error`, see https://golang.org/pkg/text/template/#Template.Option but I think it's better to assume we're using the right option instead of putting the "gohai missing case" behind an `if` statement in every template using it.